### PR TITLE
Ta not required flow

### PR DESCRIPTION
**Pull request by simon-the-sloth**

### CODE REVIEW

**Comment by robertavram** [comment]
@simon-the-sloth please list the issues that are being solved here

**Comment by simon-the-sloth** [comment]
Issue 164 will be fixed by this. (Ticket will be closed by frontend pr)

**Comment by ewheeler** [comment]
@simon-the-sloth please link issues rather than just writing the number

For example: unicef/etools-issues#164

**Comment by simon-the-sloth** [comment]
@ewheeler The ticket contains frontend work too and that's why I don't wanted to move by this PR.

**Comment by ewheeler** [comment]
@simon-the-sloth ok, please explain that in the comments. we can add a label to signify this as well if you'd like

File: EquiTrack/t2f/urls.py [hunk 1]
```diff
@@ -59,7 +59,7 @@
 travel_list_patterns = patterns(
     '',
     url(r'^$', travel_list, name='index'),
-    url(r'^(?P<transition_name>save_and_submit)/$', travel_list_state_change, name='state_change'),
+    url(r'^(?P<transition_name>save_and_submit|mark_as_completed)/$', travel_list_state_change, name='state_change'),
```
Comment (simon-the-sloth) [inline EquiTrack/t2f/urls.py]: It is. For the new flow, this should be possible.

**Comment by simon-the-sloth** [comment]
@ewheeler In the future if it fixes only the backend part of a task, I'll refer to the ticket and the frontend pr if it's possible.

**Pull request merged**